contracts/ParsecBridge.sol

```diff
@@ -189,6 +189,7 @@ contract ParsecBridge {
     Slot storage slot = slots[_slotId];
```

Comment:
I would add the check here, so that bet could be done by _owner only

Reply:
i think this is addressed by using `tx.origin`, right?